src/Storefront/Resources/app/storefront/src/plugin/cookie/cookie-accept-all.plugin.js

```diff
@@ -0,0 +1,85 @@
+import Plugin from 'src/plugin-system/plugin.class';
```

Comment:
Can we put this functionality for "Accept all" in the existing `cookie-permission.plugin.js`? The logic for `_hideCookieBar` seems to be available already in this plugin. What do you think @bschulzebaek?

Reply:
Sure, this would prevent a lot of duplicated code. 👍